### PR TITLE
Add placeholder to search input, trying to make clear it's DGS signs

### PR DESCRIPTION
**Pull request by alvarofernandoms**
Related with #234

### CODE REVIEW

File: priv/gettext/de/LC_MESSAGES/default.po [hunk 1]
```diff
@@ -1255,7 +1255,13 @@ msgid "New videos"
...
+#: lib/sign_dict_web/controllers/entry_controller.ex:44
 #: lib/sign_dict_web/templates/entry/latest.html.eex:5
 msgid "Recently created videos"
 msgstr "Zuletzt angelegte Videos"
+
+#, elixir-format
+#: lib/sign_dict_web/templates/page/index.html.eex:31
+#: lib/sign_dict_web/templates/shared/searchbar.html.eex:6
+msgid "Search a word to see the GSL sign"
+msgstr "Suche ein Wort um das DGS Schild zu sehen"
```
Comment (bitboxer) [inline priv/gettext/de/LC_MESSAGES/default.po]:
```suggestion
msgstr "Suche ein Wort um die Gebärde in DGS zu sehen"
```

**Comment by bitboxer** [comment]
@alvarofernandoms 👋 do you update that PR or should I merge and update it later on?

**Comment by alvarofernandoms** [comment]
Hello, @bitboxer. I'll fix in this PR.

**Comment by bitboxer** [comment]
Ah damn. I updated the text just before. I will fix that part tomorrow. Don't worry about it :wink:

**Comment by bitboxer** [comment]
Thanks again for your PR. It is merged and will be deployed now

**Pull request merged**